version/mongodb_3_6_3sh

```diff
@@ -2,6 +2,8 @@
 MONGO_VERSION=3.6.3
 echo "================= Installing Mongodb 3.6.3 ==================="
 echo "[mongodb-org-3.6]
+sudo service mongod restart
```

Comment:
why is this required?

Reply:
ther is an error like mongodb will not be started because the binary was not found at /usr/bin/mongod.


https://app.shippable.com/github/Shippable/jobs/all_x86_64_CentOS_7_prep/builds/5acf598cb356560700a2a565/console

Reply:
i dont understand how this will solve the problem. 
the error says that mongodb binary was not found and the fix here is to restart mongo service.